streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -172,15 +169,18 @@ private void close(final boolean clean) {
             transitionTo(State.CLOSING);
         } else {
             if (state() == State.RUNNING) {
-                if (clean)
```

Comment:
Not sure how this slipped by checkstyle?

Reply:
I think checkstyle does not enforce it... Even if I think it should...